bundles/mapping/mapmodule/AbstractMapModule.js

```diff
@@ -1068,6 +1070,15 @@ Oskari.clazz.define(
             var evt = Oskari.eventBuilder('AfterMapMoveEvent')(lonlat.lon, lonlat.lat, this.getMapZoom(), this.getMapScale(), creator);
             sandbox.notifyAll(evt);
         },
+
+        notifyTourEvent: function (status, cancelled = false) {
```

Comment:
Should we switch canceled to completed and calculate canceled based on status.steps !== status.step?

Reply:
It now gets completed if the current step === last step, and cancelled if the animation event doesn't get to finish (for example if user moves map).